masterlist.yaml

```
@@ -12469,6 +12469,34 @@ plugins:
   - name: 'Falskaar.esm'
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/2057/' ]
     msg:
+      - <<: *alsoUseX
+        subs: [ '[Falskaar - Addons and Patches](https://www.nexusmods.com/skyrimspecialedition/mods/19454/)' ]
```

```suggestion
        subs: [ '[Falskaar - Bug Fixes](https://www.nexusmods.com/skyrimspecialedition/mods/19454/)' ]
```
Just to make it clear what plugin is being recommended.